Vagrantfile

```diff
@@ -16,42 +16,65 @@ File.open("inventory.ini", "w") do |file|
 end
 
 Vagrant.configure("2") do |config|
-  # setup control node
+  # --- Control Node ---
   config.vm.define "controller" do |control|
-    control.vm.box = "bento/ubuntu-24.04"
-    control.vm.box_version = "202404.26.0"
+    control.vm.box = "bento/ubuntu-22.04"
+    control.vm.box_version = "202404.23.0"
     control.vm.hostname = "controller"
     config.vm.network "private_network", ip: "192.168.56.10"
```

Comment:
Is there a reason this is config.vm.network not control.vm.network?

Reply:
oh huh no, probably made a mistake when updating the code